libs/community/langchain_community/vectorstores/surrealdb.py

```diff
@@ -12,6 +12,16 @@
 from langchain_core.vectorstores import VectorStore
 
 
+try:
```

Comment:
this should stay in init, otherwise i think it will block imports of any vectorstore (since this is imported in init)

Reply:
Ah good callout, I will move the import back